docs/appendices/release-notes/unreleased.rst

```diff
@@ -93,6 +93,9 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
+- Fixed inefficient join optimizations on hash and nested-loop joins when
+  table statistics were giving misleading indications.
```

Comment:
Wouldn't this describe it more concrete?
```suggestion
  table statistics aren't available.
```

Reply:
👍